diff -up gnome-initial-setup-3.14.4/gnome-initial-setup/pages/account/gis-account-page-local.c.debug gnome-initial-setup-3.14.4/gnome-initial-setup/pages/account/gis-account-page-local.c
--- gnome-initial-setup-3.14.4/gnome-initial-setup/pages/account/gis-account-page-local.c.debug	2015-07-15 15:54:09.023635902 -0400
+++ gnome-initial-setup-3.14.4/gnome-initial-setup/pages/account/gis-account-page-local.c	2015-07-15 15:54:28.502817436 -0400
@@ -192,7 +192,6 @@ prepopulate_account_page (GisAccountPage
         /* FIXME: collect information from more than one account
          * and present at least the pictures in the avatar chooser
          */
-        g_print ("Got profile information: Name '%s' Picture '%s'\n", name, picture);
         break;
       }
     }
