diff -up gnome-initial-setup-3.14.4/gnome-initial-setup/pages/timezone/gis-bubble-widget.ui.style gnome-initial-setup-3.14.4/gnome-initial-setup/pages/timezone/gis-bubble-widget.ui
--- gnome-initial-setup-3.14.4/gnome-initial-setup/pages/timezone/gis-bubble-widget.ui.style	2015-07-15 15:49:10.822147169 -0400
+++ gnome-initial-setup-3.14.4/gnome-initial-setup/pages/timezone/gis-bubble-widget.ui	2015-07-15 15:49:22.710233673 -0400
@@ -5,6 +5,9 @@
     <child>
       <object class="GtkAspectFrame" id="aspect_frame">
         <property name="visible">True</property>
+        <style>
+          <class name="gis-bubble-widget" />
+        </style>
         <child>
           <object class="GtkBox" id="box">
             <property name="visible">True</property>
@@ -29,9 +32,6 @@
               </object>
             </child>
           </object>
-          <style>
-            <class name="gis-bubble-widget" />
-          </style>
         </child>
       </object>
     </child>
